skaffold.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: skaffold/v4beta11
+kind: Config
+metadata:
+  name: browsertrix
+build:
+  artifacts:
+    # only put the backend in here since it's quite easy now
+    # to run frontend locally and get hot reloading with yarn
+    - image: docker.io/webrecorder/browsertrix-backend
+      context: backend
+      docker:
+        dockerfile: Dockerfile
+      sync:
+        manual:
+        - src: 'btrixcloud/**/*'
+          dest: .
+portForward:
+- resourceType: service
+  resourceName: browsertrix-cloud-backend
+  # assumes you are doing local dev in `default` namespace
+  # if you aren't you need to change this manually
+  namespace: default
+  port: 8000
+  localPort: 8000
+deploy:
+  helm:
+    releases:
+      - name: btrix
+        chartPath: chart
+        valuesFiles:
+          - chart/values.yaml
```

Comment:
Since this is for local development, we may also want to add `chart/local.yaml` here, since our local development documentation instructs user to add development overrides in that file rather than in `chart/values.yaml` specifically.

`./btrix setup` from the repo root directory will copy the example local config file to the expected `chart/local.yaml`, perhaps there's a way to automate creating it if it doesn't yet exist so that Skaffold doesn't complain. At the least, we could point out that option in the documentation as a step in the new (excellent!) hot reloading docs.

Reply:
Makes sense! Just added the `local.yaml` file to skaffold config. 

I think that removes the need for changes to `./btrix setup` but lmk if I misunderstood you.